bindings/swift/CHANGELOG.md

```diff
@@ -0,0 +1,25 @@
+# Changelog
+
+All notable changes to this project will be documented in this file.
+
+The format is based on [Keep a Changelog](https://keepachangelog.com/en/1.0.0/),
+and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0.html).
+
+<!-- ## Unreleased - YYYY-MM-DD
+
+### Added
+
+### Changed
+
+### Deprecated
+
+### Removed
+
+### Fixed
+
+### Security -->
+
+## 0.1.1 - 20XX-XX-XX
+
+### Removed
+- iota_clear_listeners in favour of sendMessage("ClearListeners");
```

Comment:
Not released yet so don't put any diff, just say something like `First RC for the Swift bindings of the wallet.`